source/access/generating_keys_with_putty.rst

```diff
@@ -1,4 +1,4 @@
-.. _generating keys windows:
+.. _generating keys putty:
```

Comment:
All links to `generating keys windows` should be updated to `generating keys putty` as well. This is what I found in `development`
```
source/access/getting_access.rst
55:- :ref:`Windows <generating keys windows>` and

source/access/account_request.rst
42:- :ref:`Windows <generating keys windows>` and

source/access/access_from_multiple_machines.rst
14:   pages <generating keys windows>`.
17:   (:ref:`Windows <generating keys windows>`, :ref:`macOS/OS

source/access/messed_up_keys.rst
27:      :ref:`Windows <generating keys windows>` and :ref:`macOS

source/access/creating_a_ssh_tunnel_using_openssh.rst
10:   on generating keys <generating keys windows>`.

source/access/data_transfer_using_winscp.rst
21:program <generating keys windows>`, can :ref:`launch terminal

source/access/text_mode_access_using_putty.rst
10:<generating keys windows>` and have an approved account on the VSC clusters.

source/access/using_pageant.rst
13:PuTTY <generating keys windows>` to find out how to

source/access/windows_client.rst
13:<generating keys windows>`.
```

Reply:
ah yes, but not all of them need to be updated.

Reply:
ok, done, hopefully all links work now...
I also added a "steps to get access" section at the beginning, because several users complained about the messy structure.